src/components/SensorOverlay/stories/basic.story.mdx

```diff
@@ -0,0 +1,273 @@
+`timeseriesIds` prop SensorOverlay fetches meta information (name, description, etc) for all timeseries in the list and then constantly fetches latest data (Datapoint) for each timeserie with interval provided in `refreshInterval` (5 seconds by default).
+
+**NOTE:** The component should have `ClientSDKProvider` as a parent component in react component tree.
+
+### Basic usage:
+
+Imports:
+
+```jsx
+import 'antd/dist/antd.css';
+
+import React from 'react';
+import { SensorOverlay } from '@cognite/gearbox';
+
+CustomizedDiv = () => (
+  <div style={{ width: '100%', height: '160px', background: '#EEE' }} />
+);
+```
+
+<Preview>
+  <Story name="Basic usage"><SensorOverlay timeseriesIds={[8681821313339919]}><CustomizedDiv/></SensorOverlay></Story>
+</Preview>
+
+## Available props
+
+<Props of={SensorOverlayWithoutCustomize} />
+
+### Types
+
+#### SensorPosition
+ This type can be imported from @cognite/gearbox:
+```jsx
+import { SensorPosition } from '@cognite/gearbox';
+```
+Definition:
+```jsx
+interface SensorPosition {
+  left: number;
+  top: number;
+  pointer: {
+    left: number;
+    top: number;
+  };
+}
+```
+The interface defines position for tag and pointer of a sensor relatively to the size of wrapped content provided as `children`. `left` and `top` are coordinates of the tag whereas `pointer` object contains coordinates of the pointer.
+These coordinates are float numbers from 0 to 1.
+#### SensorMinMaxRange
+This interface defines minimum and maximum values for a datapoint of a timeserie.
+The type can be imported from @cognite/gearbox:
+```jsx
+import { SensorMinMaxRange } from '@cognite/gearbox';
+```
+Definition: 
+```jsx
+interface SensorMinMaxRange {
+  min?: number;
+  max?: number;
+}
+```
+
+#### SensorDatapoint
+This type can be imported from `@cognite/gearbox`:
+```jsx
+import { SensorDatapoint } from '@cognite/gearbox';
+```
+
+Definition:
+```jsx
+export interface SensorDatapoint {
+  isString: boolean;
+  value: number | string;
+  timestamp: Date;
+}
+```
+## Examples
+
+### With Many Sensors
+
+```jsx
+ const timeseriesIds = [
+    8681821313339919,
+    4965555138606429,
+    1762612637163055,
+    7108578362782757,
+  ];
+```
+<Preview>
+  <Story name="With Many Sensors">
+    <SensorOverlay timeseriesIds={timeseriesIds}><CustomizedDiv/></SensorOverlay>
+  </Story>
+</Preview>
+
+### Default Position And Color
+
+```jsx
+const colorMap = { [8681821313339919]: '#33AA33' };
+
+const defaultPositionMap = {
+  [8681821313339919]: {
+    left: 0.5,
+    top: 0.2,
+    pointer: {
+      left: 0.4,
+      top: 0.8,
+    },
+  },
+};
+```
+
+<Preview>
+  <Story name="Default Position And Color">
+    <SensorOverlay timeseriesIds={timeseriesIds} colorMap={colorMap} defaultPositionMap={defaultPositionMap}><CustomizedDiv/></SensorOverlay>
```

Comment:
alignment

Reply:
For better showing in the code snippet in docs, we have to keep it like this (inline)